src/Functions/monthName.cpp

```diff
@@ -0,0 +1,83 @@
+#include <Functions/dateName.h>
```

Comment:
We do not prefer to move functions into .h file, consider to get `dateName` function from function factory and execute it.

Reply:
Done. Please take a look.